docs/cudf/source/user_guide/performance_comparisons.ipynb

```diff
@@ -0,0 +1,1699 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    os.system("/sbin/sysctl vm.drop_caches=3")
</pre><p>This deserves a comment explaining why this step is needed. Also please use <code>subprocess</code> instead of <code>os.system</code> .  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/12595/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
I saw @GregoryKimball's comment above -- but I'm not sure that this cache will affect in-memory benchmarks. I think it's only relevant if reading from disk. If this notebook focuses on memory I/O then don't worry about executing that step.

Reply:
Right, dropped this for this notebook. I will addressed that as part of another notebook dedicated for IO